backend-services/user-verification-service/src/main/java/org/cardano/foundation/voting/service/discord/DefaultDiscordUserVerificationService.java

```diff
@@ -428,9 +428,15 @@ private Either<Problem, IsVerifiedResponse> handleKeriVerification(DiscordCheckV
 
         // Step 1: Check if OOBI is already registered
         Either<Problem, String> oobiCheckResult = keriVerificationClient.getOOBI(oobi, 1);
-
         if (oobiCheckResult.isRight()) {
-            log.info("OOBI already registered: {}", oobiCheckResult);
+            log.info("OOBI already registered: {}", oobiCheckResult.get());
+
+            // Step 1.1:Update key state
+            Either<Problem, Boolean> keyStateUpdateResult = keriVerificationClient.updateAndVerifyKeyState(walletId, 60);
```

Comment:
but I guess for edge case reasons it's OK if we verify here. (so we've resolved the OOBI, but for some reason the user is registering again, e.g. in case of failure on the Java side completing but Python side completed - they might re-register after another key rotation)